package.json

```diff
@@ -32,7 +32,8 @@
     "mongoose": "^4.11.6",
     "mongoose-query": "^0.3.0",
     "ms": "^2.0.0",
-    "rand-token": "0.3.0"
+    "rand-token": "0.3.0",
+    "request": "2.85.0"
```

Comment:
This is not needed actually, because we already use axios.

Reply:
I have fixed this in the latest commit, now axios is used throughout the functionality.
I would still like to test that all works well against an owncloud instance.